l10n/gitahead_ja.ts

```diff
@@ -313,12 +313,12 @@
 <context>
     <name>Application</name>
     <message>
-        <location filename="../src/app/Application.cpp" line="390"/>
+        <location filename="../src/app/Application.cpp" line="393"/>
```

Comment:
For my understanding: Are these line numbers important (i.e. if they don't match anymore, then the translation is broken)? Or did you just update that so that it's 100% perfect/clear even though not strictly necessary?

Reply:
They aren't so important. I did this just to ensure that the translation is synchronized with current master.

Reply:
Thanks, then I'll ignore / not review those.